src/commands/list_folder.ts

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable no-console */
+import { arweave } from 'ardrive-core-js';
+import { cliWalletDao, CLI_APP_NAME, CLI_APP_VERSION } from '..';
+import { ArDrive, ArDriveAnonymous } from '../ardrive';
+import {
+	ArFSDAO,
+	ArFSDAOAnonymous,
+	ArFSPrivateFileOrFolderData,
+	ArFSPublicFileOrFolderData,
+	FolderHierarchy
+} from '../arfsdao';
+import { CLICommand } from '../CLICommand';
+import { CommonContext } from '../CLICommand/common_context';
+import { ArDriveCommunityOracle } from '../community/ardrive_community_oracle';
+import { ParentFolderIdParameter, SeedPhraseParameter, WalletFileParameter } from '../parameter_declarations';
+
+new CLICommand({
+	name: 'list-folder',
+	parameters: [ParentFolderIdParameter, SeedPhraseParameter, WalletFileParameter],
+	async action(options) {
+		const context = new CommonContext(options, cliWalletDao);
+		const wallet = await context.getWallet().catch(() => null);
+		const folderId = context.getParameterValue(ParentFolderIdParameter);
+		let folder;
+		let mergedData: (ArFSPrivateFileOrFolderData | ArFSPublicFileOrFolderData)[];
+
+		if (!folderId) {
+			console.log(`Folder id not specified! ${folderId}`);
```

Comment:
Probably don't need to show them the unspecified variable :D

Reply:
Removed at dd136424c231d3b047a5494e6114435cb356dd6f